dev/how-to/maintain/tidb-in-kubernetes/tidb-cluster.md

```diff
@@ -19,21 +19,20 @@ chartVersion="v1.0.0-beta.3"
 
 在 GKE 上，本地 SSD 卷默认大小限制为 375 GB，性能比永久性磁盘要差。
 
-为提高性能，你必须保证以下两点：
+为提高性能，必须保证以下两点：
 
-* 安装 Linux Guest Environment，只能用于 Ubuntu 系统，不能用于 COS。
+* 安装 Linux Guest Environment，只能用于 Ubuntu 系统，不能用于 Container-Optimized OS。
 * 确保 SSD 挂载选项中包含 `nobarrier`。
 
-我们提供一个解决上述问题的 [Daemonset](https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/gke/local-ssd-optimize.yaml)。还提供一个解决上述性能问题并且将多块 SSD 盘配置成一个 LVM 的
-[Daemonset](https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/gke/local-ssd-provision/local-ssd-provision.yaml)。通过 Terraform 部署的时候会自动安装。
+[Daemonset](https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/gke/local-ssd-optimize.yaml) 提供了一个解决上述性能问题的方案，并且将多块 SSD 盘配置成一个 LVM。通过 Terraform 部署的时候会进行自动安装。
```

Comment:
I'm afraid this is not the original meaning. Please check.

Reply:
Please refer to https://github.com/pingcap/tidb-operator/pull/612 for latest update.
I have ported that change to docs.

Reply:
I have already updated this part according to the English version. See the commit below.